File: API/Models/UploadedFiles.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    [Table("UploadedFiles", Schema = "dbo")]
    public class UploadedFiles
    {
        [Key]
        public int Id { get; set; }
        [Required]
        [MaxLength(50)]
        public string CustomerId { get; set; } = null!;
        [MaxLength(100)]
        public string? SupplierName { get; set; }
        [Required]
        public string FileName { get; set; } = null!;
        public string? FilePath { get; set; }
        [MaxLength(50)]
        public string? ProcessType { get; set; }
        public DateTime UploadedTime { get; set; }
        public int IsProcess { get; set; }=0;
        public int HeaderId { get; set; }
        public int LineIdStart { get; set; }
        public int LineIdEnd { get; set; }
    }
}

File: invoice/src/app/_model/uploadedFiles.ts
export class UploadedFiles {
    constructor(
        public id: number,
        public customerId: string,
        public supplierName: string,
        public fileName: string = '',
        public filePath: string,
        public processType?: string,
        public uploadedTime: string = '',
        public isProcess: number = 0,
        public headerId: number = 0,
        public lineIdStart: number = 0,
        public lineIdEnd: number = 0,
        public invoiceNumber: string = '',
        public invoiceDate: string = ''
    ) { }
}